site/pages/home/blog-slice/blog-entry/index.js

```diff
@@ -1,30 +1,43 @@
-import React, { PropTypes } from 'react';
+// @flow
+
+import React from 'react';
 import styles from './style.css';
 
-const BlogEntry = ({
-  category,
-  title,
-  url,
-  authorName,
-  authorTitle,
-}) => {
+type Author = {|
+  bio: string,
+  displayName: string,
+|}
```

Comment:
I would normally go for normal object types vs. strict unless there's a reason to, as it saves modifying the typedefs in future. I used strict in the routes as it's config so changes are more intentional there.